### ISSUE TITLE
initUser: Cannot read property 'NaN' of undefined

### ISSUE THREAD

**Issue by tommoor** [yjs/y-websocket#33]
**Checklist**

* [x] Are you reporting a bug? Use github issues for bug reports and feature requests. For general questions, please use https://discuss.yjs.dev/
* [x] Try to report your issue in the correct repository. Yjs consists of many modules. When in doubt, report it to https://github.com/yjs/yjs/issues/

**Describe the bug**

Exception `TypeError: Cannot read property 'NaN' of undefined` in the JS console (see screenshots).

**To Reproduce**

After including `setUserMapping` on the client, like so:

```javascript
const userData = new Y.PermanentUserData(ydoc);
userData.setUserMapping(ydoc, ydoc.clientID, user.id);

provider = new WebsocketProvider(
  "ws://localhost:1234",
  roomname,
  ydoc,
  {
    resyncInterval: 10 * 1000,
  }
);
```
 
I am regularly receiving this error in the console – I believe it's when rejoining a previously joined websocket. I can't find documentation for `PermanentUserData`, so could be using it wrong somehow? If you need additional context let me know what I can grab, it's happening enough that I can reproduce on demand.

**Screenshots**

![image](https://user-images.githubusercontent.com/380914/95530422-ac912280-0992-11eb-96a6-636c270c5e45.png)

![image](https://user-images.githubusercontent.com/380914/95530440-b9ae1180-0992-11eb-991b-aedf2f46a252.png)

**Environment Information**
 - Browser: Chrome latest
 - version: yjs@13.4.0, y-websocket@1.3.3

**Comment by dmonad** [comment]
Hi @tommoor 

`userData.setUserMapping(ydoc, ydoc.clientID, user.id);` expects a mapped string value. Is `user.id` a string?

**Comment by dmonad** [comment]
Oh.. I see that decoder.pos is `NaN`. I'm not sure how this can happen.. 

I can't reproduce the problem at the moment because the demo application is working for me.

Are you maybe connecting to a public room?

**Comment by dmonad** [comment]
I was able to reproduce the issue. I'm working on a fix.

**Comment by dmonad** [comment]
The problem resulted from some refactoring I did some time ago. The DS-decoder expected a lib0/Decoder, not an Uint8Array. Not sure why typescript didn't catch it.. Thanks for noticing!

I fixed the issue in yjs@13.4.1.
